ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/decoded-simulation/decoded-simulation.tsx

```diff
@@ -11,8 +12,35 @@ import { useI18nContext } from '../../../../../../../../hooks/useI18nContext';
 import { SignatureRequestType } from '../../../../../../types/confirm';
 import { useConfirmContext } from '../../../../../../context/confirm';
 import StaticSimulation from '../../../shared/static-simulation/static-simulation';
-import NativeValueDisplay from '../native-value-display/native-value-display';
 import TokenValueDisplay from '../value-display/value-display';
+import NativeValueDisplay from '../native-value-display/native-value-display';
+
+export const getStateChangeToolip = (
+  stateChangeList: DecodingDataStateChanges | null,
+  stateChange: DecodingDataStateChange,
+  t: ReturnType<typeof useI18nContext>,
+): string | undefined => {
+  if (stateChange.changeType === DecodingDataChangeType.Receive) {
+    if (
+      stateChangeList?.some(
+        (change) =>
+          change.changeType === DecodingDataChangeType.Listing &&
+          change.assetType === TokenStandard.ERC721,
+      )
+    ) {
+      return t('signature_decoding_list_nft_tooltip');
+    }
+    if (
+      stateChange.assetType === TokenStandard.ERC721 &&
```

Comment:
Q: 

we check the assetType on stateChange here and the assetType on the change of the list item above. Did we want to keep this as is or update to be consistent?

Reply:
Above is filter applied to whole list of state changes, and here we are checking seet of single state change.

Reply:
right. Thanks for confirming this is expected!